extensions/amp-form/0.1/amp-form.js

```diff
@@ -924,15 +924,18 @@ export class AmpForm {
   /**
    * Transition the form to the submit success state.
    * @param {!JsonObject} result
-   * @param {!JsonObject} eventData
+   * @param {?JsonObject=} opt_eventData
```

Comment:
Nit: This should be `!JsonObject=` unless you want to allow passing `null`.

Reply:
We do want to pass `null` if that is the value for `response.body`